x-pack/plugins/stack_connectors/server/connector_types/bedrock/index.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  SubActionConnectorType,
+  ValidatorType,
+} from '@kbn/actions-plugin/server/sub_action_framework/types';
+import { GeneralConnectorFeatureId } from '@kbn/actions-plugin/common';
+import { urlAllowListValidator } from '@kbn/actions-plugin/server';
+import { ValidatorServices } from '@kbn/actions-plugin/server/types';
+import { assertURL } from '@kbn/actions-plugin/server/sub_action_framework/helpers/validators';
+import { BEDROCK_CONNECTOR_ID, BEDROCK_TITLE } from '../../../common/bedrock/constants';
+import { BedrockConfigSchema, BedrockSecretsSchema } from '../../../common/bedrock/schema';
+import { BedrockConfig, BedrockSecrets } from '../../../common/bedrock/types';
+import { BedrockConnector } from './bedrock';
+import { renderParameterTemplates } from './render';
+
+export const getConnectorType = (): SubActionConnectorType<BedrockConfig, BedrockSecrets> => ({
+  id: BEDROCK_CONNECTOR_ID,
+  name: BEDROCK_TITLE,
+  Service: BedrockConnector,
+  schema: {
+    config: BedrockConfigSchema,
+    secrets: BedrockSecretsSchema,
+  },
+  validators: [{ type: ValidatorType.CONFIG, validator: configValidator }],
+  supportedFeatureIds: [GeneralConnectorFeatureId],
+  minimumLicenseRequired: 'enterprise' as const,
+  renderParameterTemplates,
+});
+
+export const configValidator = (
+  configObject: BedrockConfig,
+  validatorServices: ValidatorServices
+) => {
+  try {
+    assertURL(configObject.apiUrl);
+    urlAllowListValidator('apiUrl')(configObject, validatorServices);
```

Comment:
so i was playing around with the signature yestrerday and discovered that i can omit `method`, `service` and `region` and still have successful results. 

<img width="660" alt="Screenshot 2023-09-26 at 10 02 24 AM" src="https://github.com/elastic/kibana/assets/6935300/a483d2bb-d409-4ae7-993b-70fa281eb7aa">

Looks like they can correctly infer the values from `RequestSigner.matchHost()`.Therefore, I think it would be ok to omit the region from the config. WDYT?

FYI, I still need to send `host`, it seems the `RequestSigner.createHost()` does not get us what we need, here is the error:
```
Message: Unexpected API Error: ERR_TLS_CERT_ALTNAME_INVALID - Hostname/IP does not match certificate's altnames: Host: .us-east-1.amazonaws.com. is not in the cert's altnames: DNS:bedrock.us-east-1.amazonaws.com
```

Reply:
I think if we can omit the region from the connector config, that'd be great. Less chance of user error entering a region that doesn't match the api url